apps/admin-gui/src/app/shared/components/dialogs/create-vo-dialog/create-vo-dialog.component.html

```diff
@@ -0,0 +1,53 @@
+<h1 mat-dialog-title>{{"DIALOGS.CREATE_VO.TITLE" | translate}}</h1>
+<div class="{{theme}}">
+  <mat-spinner class="ml-auto mr-auto" *ngIf="loading"></mat-spinner>
+</div>
+<div class="{{theme}}" *ngIf="!loading">
+  <div mat-dialog-content class="dialog-container">
+
+    <mat-form-field>
+      <input
+        matInput
+        required
+        placeholder="{{'DIALOGS.CREATE_VO.FULL_NAME' | translate}}"
+        [(ngModel)]="fullName">
+      <mat-error>
+        {{'DIALOGS.CREATE_VO.FILL_VALUE' | translate}}
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field>
+      <input
+        matInput
+        required
+        placeholder="{{'DIALOGS.CREATE_VO.SHORT_NAME' | translate}}"
+        [(ngModel)]="shortName">
+      <mat-error>
+        {{'DIALOGS.CREATE_VO.FILL_VALUE' | translate}}
+      </mat-error>
+    </mat-form-field>
+
+    <mat-error *ngIf="!isValidFullName() && fullName !== ''">
+      {{'DIALOGS.CREATE_VO.FILL_FULL_NAME' | translate}}
+    </mat-error>
+
+    <mat-error *ngIf="!isValidShortName() && shortName !== ''">
+      {{'DIALOGS.CREATE_VO.FILL_SHORT_NAME' | translate}}
+    </mat-error>
+  </div>
+  <div mat-dialog-actions>
+    <button
+      mat-flat-button
+      class="ml-auto"
+      (click)="onCancel()">
+      {{'DIALOGS.CREATE_GROUP.CANCEL' | translate}}
+    </button>
+    <button
+      mat-flat-button
+      class="ml-2"
+      color="accent"
+      [disabled]="!isValidShortName() || !isValidFullName()"
+      (click)="onSubmit()">
+      {{'DIALOGS.CREATE_GROUP.CREATE' | translate}}
+    </button>
```

Comment:
If you want, you can add tooltip to disabled submit button, so user will know why it is disabled. Although it is kinda obvious here...